library/src/main/java/com/kizitonwose/calendarview/ui/CalendarAdapter.kt

```diff
@@ -97,15 +99,21 @@ internal class CalendarAdapter(
         }
 
         fun setupRoot(root: ViewGroup) {
-            root.setPaddingRelative(
+            ViewCompat.setPaddingRelative(root,
                 calView.monthPaddingStart, calView.monthPaddingTop,
                 calView.monthPaddingEnd, calView.monthPaddingBottom
             )
             root.layoutParams = ViewGroup.MarginLayoutParams(LP.WRAP_CONTENT, LP.WRAP_CONTENT).apply {
                 bottomMargin = calView.monthMarginBottom
                 topMargin = calView.monthMarginTop
-                marginStart = calView.monthMarginStart
-                marginEnd = calView.monthMarginEnd
+
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
```

Comment:
This check is no longer needed since it'll always be `true`.

Reply:
Sure thing. However, it is necessary in order to work properly on api 15 with library override mechanism. So I suggest you keep it. What do you think ?

Reply:
You know what? Let's proceed with the min SDK change, guess you really want it. I'll close this PR and just reopen the other one.

Reply:
Thanks!